data/replaced_tags.txt

```diff
@@ -92,11 +92,6 @@ natural=forest    : natural=wood
 natural=shrubbery : natural=scrub
 cliff=yes         : natural=cliff
 
-desert=sand : desert=erg
```

Comment:
I don't know where this key is coming from (undocumented in the OSM wiki), but it seems its really used for some bigger deserts....